### PR TITLE
use electron-builder to package the app

### PR DESCRIPTION
**Pull request by jffmarker**
This should allow build pipelines to run `npm run pack` and ZIP the 'dist/win-unpacked', 'dist/linux-unpacked', or 'dist/mac' directories to create portable packages.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -20,3 +20,4 @@ jobs:
       - run: npm ci
       - run: npm run lint
       - run: npm run build
+      - run: npm run pack
```
[the remark this answers is not in the document]
Comment (jffmarker) [reply .github/workflows/ci.yaml:23]: "dist" and "pack" are the default words in the [quick start](https://www.electron.build/#quick-setup-guide)... but I don't particularly love or hate them. Would you prefer any of these (as examples)?
1. package
1. apppackage
1. build:pack

[the remark this answers is not in the document]
Comment (calebmshafer) [reply .github/workflows/ci.yaml:23]: Interesting that it's the recommendation.  My vote would be for `build:pack` instead but could live with it the way it is now too...

**Comment by jffmarker** [comment]
> The workflow for the build will be to use `npm run build` instead of the `npm run build:prod` script. I'd like to just remove the difference between the two but that would make the initial build slower...
> 
> Otherwise, this looks good to me. I tried it out on Mac and worked like a charm.

@calebmshafer, until then, should I clean and then run `build:prod`... or just switch `build` to `build:prod`?

**Comment by calebmshafer** [comment]
@jffmarker mind switching the CI to use `build:prod` instead of `build`?

**Pull request merged**